README.rst

```diff
@@ -94,56 +93,29 @@ Create a test file in the tests folder
             def empty_method():
                 pass
 
-Create a Dockerfile-splunk file
-
-.. code:: Dockerfile
-
-    ARG SPLUNK_VERSION=latest
-    FROM splunk/splunk:$SPLUNK_VERSION
-    ARG SPLUNK_APP=TA_UNKNOWN
-    ARG SPLUNK_APP_PACKAGE=package
-    COPY deps/apps /opt/splunk/etc/apps/
-
-    COPY $SPLUNK_APP_PACKAGE /opt/splunk/etc/apps/$SPLUNK_APP
-
-
-Create a docker-compose.yml update the value of SPLUNK_APP
-
-.. code:: yaml
-
-    version: "3.7"
-    services:
-    splunk:
-        build:
-        context: .
-        dockerfile: Dockerfile-splunk
-        args:
-            - SPLUNK_APP=xxxxxxx
-        ports:
-        - "8000"
-        - "8089"
-        environment:
-        - SPLUNK_PASSWORD=Changed@11
-        - SPLUNK_START_ARGS=--accept-license
-
 Run pytest with the add-on and SA-eventgen installed and enabled in an external Splunk deployment
 
-.. code::: bash
+.. code::: python3
 
         pytest \
         --splunk-type=external \
-        --splunk-type=external \
         --splunk-host=splunk \
         --splunk-port=8089 \
         --splunk-password=Changed@11 \
         -v
 
-Run pytest with the add-on and SA-eventgen installed and enabled in docker
+Run pytest with the add-on and SA-eventgen installed and enabled in kubernetes
+
+Generate addon SPL with `ucc-gen`_.
+
+For third-party addons generate addon SPL.
 
-.. code::: bash
+Place the generated addon spl in tests/src/<addon>.spl
+
+.. code::: python3
 
         pytest \
-        --splunk-password=Changed@11 \
```

Comment:
Should param `--splunk-password` setup password for splunk? If yes I think it is not working

Reply:
For --splunk-type=kubernetes password and hec-token will be generated dynamically which will be used while executing tests.